types/src/collection.rs

```diff
@@ -27,4 +28,13 @@ pub struct Chapter {
     pub document_short_name: Option<String>,
 
     pub wordpress_id: Option<i64>,
+
+    pub chapter_type: CollectionSection,
+}
+
+#[derive(Debug, Clone, PartialEq, sqlx::Type)]
+#[sqlx(type_name = "collection_section")]
+pub enum CollectionSection {
+    Intro,
+    Body,
```

Comment:
I think we should probably be consistent with calling the second section of chapters either 'Genre' or 'Body' chapters-- if 'Body' chapters is preferred, I can rename other areas where we reference them throughout the migration code. Should I also change the mockup spreadsheet to reflect that?

Reply:
Yeah I think it should be "Body" because we don't know how future collections will be split up.

Reply:
And if you could propagate that, that'd be great thanks!

Reply:
sounds good thanks-- I changed naming across the migration code to reflect this as well as the second section of the spreadsheet mockup.